### PR TITLE
Resolve to a group of names

### LINKED ISSUE #504: Unstable reverse DNS lookups in UI
When viewing the connections of a container talking to servers with multiple A-records, the view switches the displayed server names frequently making it confusing.

I tried to record the example but I wasn't fast enough since the names displayed ended up stabilizing. Will add it later if I see it again.

### PR DESCRIPTION
**Pull request by inercia**
Fixes #504

### CODE REVIEW

File: probe/endpoint/resolver.go [hunk 1]
```diff
@@ -72,8 +72,10 @@ func (r *reverseResolver) loop() {
 		names, err := r.Resolver(request)
 		if err == nil && len(names) > 0 {
 			sort.Strings(names)
```
Comment (tomwilkie) [inline probe/endpoint/resolver.go]: Sort is not needed anymore, I think.

**Comment by tomwilkie** [comment]
Also pretty sure this is going to need an update in https://github.com/weaveworks/scope/blob/master/render/detailed_node.go#L256

File: probe/endpoint/resolver.go (hunk 1 above)
Comment (inercia) [inline probe/endpoint/resolver.go]: Done.

**Comment by tomwilkie** [comment]
LGTM!

**Pull request merged**